Big-Spender/readme.md

```diff
@@ -68,39 +69,48 @@ INSERT YOUR QUERY HERE
 **You:** Then here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+INSERT : select transaction_no, description from spends where lower(description) like '%fee%';
 ```
 
 **Farnoosh:** Hi, it's me again. It turns out we also need the transactions that have the expense area of 'Better Hospital Food'. Can you help us with that one?
 
 **You:** No worries. Here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+INSERT: SELECT * from spends join expense_areas on (expense_areas.id= spends.expense_area_id) where expense_area='Better Hospital Food'; ;
+ For transcation nr only  : SELECT transaction_no from spends join expense_areas on (expense_areas.id= spends.expense_area_id) where expense_area='Better Hospital Food'; ;
```

Comment:
nice job showing the two options. With select * what will this return? Everything from spends, everything from expense_areas, everything from both? How could specify this?